Source/WebKit/Platform/IPC/Connection.h

```diff
@@ -211,7 +240,7 @@ class Connection : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<Connec
         HANDLE handle { 0 };
 #elif OS(DARWIN)
         explicit Identifier(Handle&& handle)
-            : Identifier(handle.leakSendRight())
+            : Identifier(handle.handle.leakSendRight())
```

Comment:
handle.release/handle.leak/handle.leakSendRight 🤦

Reply:
Sad indeed. Ideally we'd have move-only semantics for MachSendRight (bug 257580), and a common interface for the darwin/win32/posix implementations, and then this wrapper class can go away.